bitnami/mysql/templates/primary/svc.yaml

```
@@ -40,6 +40,12 @@ spec:
       port: {{ .Values.primary.service.ports.mysql }}
       protocol: TCP
       targetPort: mysql
+    {{- if .Values.mysqlx.port.enabled }}
+    - name: mysqlx
+      port: 33060
```

I'd parameterize the port as it is done for the `mysql` one
```suggestion
      port: {{ .Values.primary.service.ports.mysqlx }}
```

---

Updated to use the parameter. Thanks for noticing it.